websites/docs-smoke-test/package.json

```diff
@@ -18,8 +18,12 @@
     "@commercetools-docs/gatsby-theme-code-examples": "19.3.0",
     "@commercetools-docs/gatsby-theme-constants": "19.3.0",
     "@commercetools-docs/gatsby-theme-docs": "19.4.0",
+    "@commercetools-docs/gatsby-theme-learning": "1.0.0",
+    "@commercetools-docs/gatsby-theme-sso-ui-kit": "1.0.0",
     "@commercetools-docs/ui-kit": "19.3.0",
+    "@commercetools-uikit/avatar": "15.2.1",
     "@commercetools-uikit/icons": "^15.0.0",
+    "@commercetools-uikit/radio-input": "15.2.1",
```

Comment:
Are these deps required?

Reply:
In the initial implementation of the learning plugin, I had to extra add those dependencies to make it work. But I will double check once we've solved the other issues